doc/changes/latest.rst

```diff
@@ -49,3 +49,5 @@ Changes
 
 - :bdg-dark:`Code` Use red to blue color map in the GLM reports (:gh:`4266` by `Rémi Gau`_).
 - :bdg-dark:`Code` :class:`nilearn.maskers.NiftiSpheresMasker` will throw warnings if the ``labels`` passed to it is not a :obj:`list` of :obj:`str`, or if the number of items in the list of labels does not match the number of regions in the label image (:gh:`4274` by `Rémi Gau`_).
+
+- :bdg-dark:`Code` Copy headers from user-specified image to the result of :func:`nilearn.image.math_img` (:gh:`4337` by `Himanshu Aggarwal`_).
```

Comment:
is it just a change or actually a fix?

Reply:
I was a bit confused about that, too.

But here's my thinking behind tagging it as a change:
The default behavior is still what was initially reported. We are just giving the user an option to copy the header if they want.

Reply:
OK makes sense: I got confused because it was closing an issue filed as bug, so I would have seen this as a fix.

that's fine then.